mmstudio/src/main/java/org/micromanager/internal/positionlist/TileCreatorDlg.java

```diff
@@ -364,6 +376,13 @@ public void actionPerformed(ActionEvent arg0) {
       studio.events().registerForEvents(this);
    }
 
+   @Subscribe
+   public void stuttingDown(ShutdownCommencingEvent se) {
```

Comment:
Ah!  The reason for the first line is that the actionEvent is not always fired when changing the value of the field (which is also why this field is read and the value stored when pressing the OK or Cancel buttons).  The call to activateAxisTable is superfluous and may even be dangerous since it is possible that positionLIstDlg_ is already disposed off.  I'll open another PR..